guides/common/assembly_importing-kickstart-repositories.adoc

```diff
@@ -1,18 +1,6 @@
 ifdef::context[:parent-context: {context}]
 
-[id="importing-kickstart-repositories_{context}"]
-= Importing Kickstart Repositories
-
-Kickstart repositories are not provided by the Content ISO image.
-To use Kickstart repositories in your disconnected {Project}, you must download a binary DVD ISO file for the version of {RHEL} that you want to use and copy the Kickstart files to {Project}.
-
-To import Kickstart repositories for {RHEL} 7, complete xref:importing-kickstart-repositories_rhel-7[].
-
-To import Kickstart repositories for {RHEL} 8, complete xref:importing-kickstart-repositories_rhel-8[].
-
-ifndef::satellite[]
-You can follow these procedures for {RHEL} derivatives such as Alma Linux, CentOS, Oracle Linux, and Rocky Linux, too.
-endif::[]
+include::modules/con_importing-kickstart-repositories.adoc[]
```

Comment:
Mark -- see my comment for `assembly_synchronizing-content-between-foreman.adoc`